pkg/driverbuilder/templates.go

```diff
@@ -72,25 +72,60 @@ type driverConfigData struct {
 	DeviceName    string
 }
 
-// TODO(fntlnz): these variables are still called probes because of how the driver code
-// references to them. We must change the driver code and then change here to s/PROBE_*/MODULE_*/g
-const driverConfigTemplate = `
+// XXX both PROBE and DRIVER variables are kept for now so that Driverkit is compatible with older versions.
+// they can be removed when versions from early 2022/late 2021 will not be supported anymore.
+const fillDriverConfigTemplate = `
+set -euxo pipefail
+
+DRIVER_BUILD_DIR=$1
+DRIVER_CONFIG_FILE="$DRIVER_BUILD_DIR/driver_config.h"
+
+cat << EOF > $DRIVER_CONFIG_FILE
 #pragma once
 
-#define PROBE_VERSION "{{ .DriverVersion }}"
+#define DRIVER_VERSION "{{ .DriverVersion }}"
+
+#define DRIVER_COMMIT "{{ .DriverVersion }}"
```

Comment:
Why this? Where it is used?

Reply:
https://github.com/falcosecurity/libs/blob/cc42b767e854256355a25f5a3779114b63bd056c/driver/driver_config.h.in

It entered libs with https://github.com/falcosecurity/libs/pull/39!

Reply:
Hey @leodido 

note that `DRIVER_COMMIT` is intended only for debugging purposes. 

@LucaGuerra please note that in the future `DRIVER_COMMIT` will be different from `DRIVER_VERSION`.
For example `DRIVER_COMMIT` will continue to be equal to the commit id, but  `DRIVER_VERSION` will be equal to the driver version (es 1.0.0).

Reply:
Thanks! Yes, I really didn't know what to put in `DRIVER_VERSION`. I expect that could change in the future but we can't predict it right now I think.

Reply:
Thank you, now it makes sense.

It's probably worth leaving a comment or opening an issue to remember the distinction...

Reply:
... and the comment was added!